components/borrowings/README.md

```diff
@@ -0,0 +1,20 @@
+## Borrowings
+
+Lexical borrowings or loanwords provide an interesting object of study by themselves
+(see for example [The World Loanword Database](http://wold.clld.org/)), but they
+also play a role for cognate detection algorithms, where it is typically important
+to distinguish between cognates due to borrowing and cognates due to inheritance.
+
+Conceptually, borrowings are the 
+[associative entity](https://en.wikipedia.org/wiki/Associative_entity) 
+linking two rows in the `FormTable` of a [wordlist](../../modules/Wordlist).
+Thus, CLDF provides a borrowing table, with the default description
+[`BorrowingTable-metadata.json`](BorrowingTable-metadata.json).
+
+A `null` value for the `sourceFormReference` marks missing knowledge about the source word
+(or donor language). In this case, a `Comment` should be used to describe the available
+knowledge.
+
+Often datasets will add a column specifying reliability levels of the loan assessment
+(e.g. WOLD's ["borrowed score"](http://wold.clld.org/terms#borrowed_score)),
+but no common encoding scheme for such levels has been conventionalized yet.
```

Comment:
It would still be nice to give suggestions on how to eg. implement the WoLD assessments within the CLDF standard.

Reply:
I may do that as an example. But before I can put together examples, I have to update `pycldf`.

Reply:
I guess, it depends on the use case. If I wanted to stuff all of WOLD into CLDF - e.g. as export - I'd just go with a column modeling closely what I have in the database. If I wanted to use WOLD for automated cognate detection, I'd decide on a cut-off, e.g. consider everything with a borrowed score >= 0.5 as borrowed and don't code reliability at all.

Reply:
That's fine. I noticed it as a thing I would like to see, but then I saw that it should be possible to puzzle it together from the other components, which I think was not really possible before, so it's not urgent.

Reply:
Part of what I hope to achieve with CLDF is that such transformations between various CLDF representations are easy enough (be it with existing tools such as excel or csvkit, or with tools built on `pycldf`) for the linguist to do.